klt-cert-manager/config/rbac/role.yaml

```diff
@@ -55,9 +55,16 @@ rules:
   - secrets
   verbs:
   - create
+  - list
```

Comment:
`list` and `watch` permissions are required due to the `Owns(&corev1.Secret{})` in the controller, which is needed to reconcile after the generated secret has been changed or deleted